Rakefile

```diff
@@ -1,8 +1,12 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require 'bundler/gem_tasks'
+require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
```

Comment:
* For consistency; doing things one way, is better than doing things two ways. I'd like to stick with one form of quotes unless the situation requires something else. Double quotes fits most situations so we should stick with that.
* For development speed; I don't want to have to change the quoting symbol when adding or removing interpolation to a string. That wastes time and effort.